packages/iris-grid/src/mousehandlers/IrisGridContextMenuHandler.tsx

```diff
@@ -546,6 +549,78 @@ class IrisGridContextMenuHandler extends GridMouseHandler {
       }
     }
 
+    // Expand/Collapse options
+    if (isExpandableGridModel(model) && model.isRowExpandable(modelRow)) {
+      // If there are grouped columns, then it is a rollup
+      // For rollups, the column number will be the depth minus one
+      let cellValue =
+        model.groupedColumns.length > 0
+          ? model.textForCell(model.depthForRow(modelRow) - 1, modelRow)
+          : model.textForCell(0, modelRow);
+
+      if (cellValue === '') {
+        cellValue = 'null';
+      }
```

Comment:
We're going to need better handling for `null` vs. `empty` string. Consider this snippet:
```
from deephaven.constants import NULL_INT
from deephaven import empty_table

source = empty_table(100).update_view(["ID = i == 0 ? `` :  `` + (i + 2)", "Parent = i == 0 ? null : i % 2 == 0 ? `` : `` + (i % 9)"])

result_no_orphans = source.tree(id_col = "ID", parent_col = "Parent")
result_w_orphans = source.tree(id_col = "ID", parent_col = "Parent", promote_orphans=True)
```
Right now it's showing "Expand row for 'null'" when the string value is empty:
![image](https://user-images.githubusercontent.com/4505624/216132183-11c8394f-d2db-4ad8-bab0-a8059b400cf1.png)

What you want to do is get `valueForCell`, which will return `null` if the cell is actually null, and then the empty string `''` if it's just an empty string (or the value for any other situation).
Then, if the value is `null`, just display `null`, otherwise display the value of `textForCell`.